core/cron.py

```diff
@@ -0,0 +1,24 @@
+""" This module has all core app corn jobs"""
+
+from datetime import datetime
+
+from templated_mail.mail import BaseEmailMessage
```

Comment:
why have you used this wrapper? i think we can use builtin [EmailMultiAlternatives](https://docs.djangoproject.com/en/4.1/topics/email/#sending-alternative-content-types)

moreover if its required, pls add it in requirements file as well